static/app/views/performance/table.tsx

```diff
@@ -55,14 +57,29 @@ import {
   UNPARAMETERIZED_TRANSACTION,
 } from './utils';
 
+type ColumnTitle = {
+  title: string | ReactNode;
+  tooltip?: string | ReactNode;
```

Comment:
```suggestion
  title: ReactNode;
  tooltip?: ReactNode;
```

A react node can be a string, so we can simplify this
https://reactnative.dev/docs/react-node